src/actions/AppActions.ts

```diff
@@ -343,6 +343,17 @@ export async function build() {
   return buildSucceeded;
 }
 
+export async function deployAndRunTests(fiddleName: string) {
```

Comment:
Let's not copy-paste this, but have common method with `pagePath` as parameter and in this case it'll be `${fiddleName}/test.html`

This makes it easier to integrate with other changes, e.g. https://github.com/nearprotocol/NEARStudio/pull/77

Reply:
needs rebasing and merging